app/helpers/auth_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AuthHelper
+  def current_user
```

Comment:
If this is in application_controller and marked helper_method you don't need to implement it her as well.